src/_root/RootLayout.tsx

```diff
@@ -1,19 +1,23 @@
 import React from 'react'
+import styles from './RootLayout.module.scss'
 import { Outlet } from 'react-router-dom'
-import { Navbar } from '../components'
+import { Footer, Navbar } from '../components'
 
 interface RootLayoutProps {
   children?: React.ReactNode
 }
 
-const RootLayout = ({ children } : RootLayoutProps) => {
+const RootLayout = ({ children }: RootLayoutProps) => {
   return (
-    <div className="">
-      <Navbar/>
-      <main>
+    <div className={styles.rootlayout}>
+      <div className={styles.rootlayout__navbar}>
```

Comment:
this `div` tag is not suppose to be here, it should be in <Navbar > component, look at the way you did Footer, this also should not be wrapped